src/js/components/TablePage.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import NasaTable from './NasaTable';
+
+class TablePage extends React.Component {
+    render() {
+        return (
+            <NasaTable/>
+        )
+    }
+}
+
+export default TablePage;
```

Comment:
Will you use some extra functionality in this component or it'll be just a component composition? 
In case it's used just to render components convert it into functional component

Reply:
resolved